src/OpenTherm.h

```diff
@@ -157,21 +160,25 @@ class OpenTherm
 	unsigned long setBoilerStatus(bool enableCentralHeating, bool enableHotWater = false, bool enableCooling = false, bool enableOutsideTemperatureCompensation = false, bool enableCentralHeating2 = false);
 	bool setBoilerTemperature(float temperature);
 	float getBoilerTemperature();
-    float getReturnTemperature();
-    bool setDHWSetpoint(float temperature);
-    float getDHWTemperature();
-    float getModulation();
-    float getPressure();
-    unsigned char getFault();
+	float getReturnTemperature();
+	bool setDHWSetpoint(float temperature);
+	float getDHWTemperature();
+	float getModulation();
+	float getPressure();
+	unsigned char getFault();
 
 private:
 	const int inPin;
 	const int outPin;
 	const bool isSlave;
+	unsigned int maxTransmissionWindowTime = 1150000; // 1000ms +15%
+	unsigned int minWaitTimeForStartBit = 0;
+	unsigned int maxWaitTimeForStartBit = 920000; // 850ms +15%
```

Comment:
why 850ms?

Reply:
Oops, this is a typo, I will correct it.
The documentation says: min 20ms, max 800ms, 800ms + 15% (tolerance) = 920ms